samples/custom_dispatch/xnnpack/plugin/system_plugin.c

```diff
@@ -70,9 +71,13 @@ static int fully_connected_nc_qd8_f32_qc4w_workgroup(void* params_ptr,
     size_t binding2_size0;
     size_t binding2_size1;
     size_t binding2_size2;
+    size_t threads;
```

Comment:
It is fine for now, but we might want to have a pointer to a context structure that includes the number of threads. Also, `num_threads` may be clearer.

Sometimes, a context might out-live the program's lifetime. The multi-node case is a good example, where we have to specify a coordinator server address and the server provides the communication channels.